data/scenarios/rows_scenarios.json

```diff
@@ -1534,6 +1534,55 @@
       }
     }
   },
+  {
+    "scenario": "Update Rows - Clear Value - Predecessor List",
+    "description": "Verifies that it is possible to clear a predecessor list using the update rows endpoint",
```

Comment:
It may be helpful to explicitly specify here that the API expects the empty predecessor list to be encoded as `null`.